src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle {
+    double kat1;
+    double kat2;
+
+    public double getKat1() {
+        return kat1;
+    }
+
+    public double getKat2() {
+        return kat2;
+    }
+
+    public void setKat1(double kat1) {
+        this.kat1 = kat1;
+    }
+
+    public void setKat2(double kat2) {
+        this.kat2 = kat2;
+    }
+    public int getArea(){
+        return (int) (0.5 * getKat1() * getKat2());

```

Comment:
Magic numbers should be avoided. The value '0.5' should be defined as a constant with a descriptive name.